docker/README.md

```diff
@@ -0,0 +1,20 @@
+# Docker support
+
+The files in this directory allow you to build a docker image.  The query coordinator assembly must be 
+copied to `query-coordinator-assembly.jar` in this directory before building.
+
+## Required Runtime Variables
+
+* `ZOOKEEPER_ENSEMBLE` - The zookeeper cluster to talk to, in the form of `["10.0.0.1:2181", "10.0.0.2:2818"]`
+* `ARK_HOST` - The IP address of the host of the docker container, used for service advertisements.
+* `SECONDARY_INSTANCES` - The list of all the secondary instances that this query coordinator should talk to.
```

Comment:
examples are nice; I'm trying to run the docker image with these environment vars and getting some errors. Not sure if it's failing to find zookeeper or bind its own address.
```
LivenessCheckResponder Unexpected exception on pong thread
java.net.BindException: Cannot assign requested address
```
```
ZOOKEEPER_ENSEMBLE=["172.19.1.150:2181"]
ARK_HOST=172.17.42.1
SECONDARY_INSTANCES=Alice,Bob,Cher
```